devops/provision/Dockerfile

```diff
@@ -1,137 +1,176 @@
 FROM wildme/wbia-dependencies:latest as org.wildme.wbia.provision
 
-MAINTAINER Wild Me <dev@wildme.org>
-
 # Set up xvfb for running gui doctests
 ENV DISPLAY :1
 
-# Clone IBEIS VSTS repositories
-RUN git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-brambox          /wbia/brambox \
- && git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-pyflann          /wbia/flann \
- && git clone --branch develop https://github.com/WildbookOrg/wbia-tpl-pyhesaff         /wbia/hesaff \
- && git clone --branch master https://github.com/WildbookOrg/wbia-plugin-flukematch     /wbia/wbia-plugin-flukematch \
- && git clone --branch master https://github.com/WildbookOrg/wbia-plugin-curvrank       /wbia/wbia-plugin-curvrank \
- && git clone --branch master https://github.com/WildbookOrg/wbia-plugin-deepsense      /wbia/wbia-plugin-deepsense \
- && git clone --branch master https://github.com/WildbookOrg/wbia-plugin-finfindr       /wbia/wbia-plugin-finfindr \
- && git clone --branch master https://github.com/WildbookOrg/wbia-plugin-kaggle7        /wbia/wbia-plugin-kaggle7 \
- && git clone --branch master https://github.com/WildbookOrg/wbia-plugin-2d-orientation /wbia/wbia-plugin-2d-orientation \
```

Comment:
This repository and code are preliminary and do not work consistently.  I have removed this plugin from the production build for the timebeing.